xml/schema/Member/MembershipType.xml

```diff
@@ -115,7 +115,8 @@
   <field>
     <name>minimum_fee</name>
     <title>membership Type Minimum Fee</title>
-    <type>decimal</type>
+    <type>varchar</type>
+    <length>512</length>
```

Comment:
Wrapping my head around the "MembershipType.minimum_fee gets saved in civicrm_price_field_value.amount and retrieved from there" - why... where..  is this triggered in the GUI?